keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,1022 @@
+>
+> To @tpepper's question of implementation, if we go with 3 symmetric releases, I would suggest using the "extra time" as a tech debt / process debt paydown phase at the beginning of each release cycle. Somewhat like how we left the milestone restriction in place at the beginning of the 1.20 release cycle. This would provide opportunity to pay down tech debt / process debt that involves large refactoring or breaking changes, the sort of work that is actively discouraged during the code freeze leading up to a release.
+>
+> I may have too narrow a view, but I have concerns that an April / August / December cadence puts undue pressure to land in August. I'm thinking of industries that typically go through a seasonal freeze in Q4. Shifting forward by a month (January / May / September) or two (February, June, October) may relieve some of that pressure, though it does cause one release to straddle Q4/Q1 in an awkward way.
+>
+> Another option is to declare Q4 what it has been in practice, a quieter time during which we're not actually going to push hard on a release, but I don't think that works as well with 3 releases vs. 4.
+
+@sebgoa:
+
+> But, generally speaking less releases (or less frequent minor releases) will also mean that each release will pack more weight, which means it will need even more testing and it will make upgrades tougher.
+>
+> With less releases developers will tend to rush their features at the last minute to "get it in" because the next one will be further apart.
+
+#### TODO Attention to tests
+
+@jberkus:
+
+>         Extra month for flakes/failures to get worse if nobody looks at them until Code Freeze
+>
+>     I think risk this can be reduced if we adjust (increase) the code freeze period.
+>
+> Better, how about we keep on top of flakes even if it's not Code Freeze?
+
+@alculquicondor:
+
+>     Better, how about we keep on top of flakes even if it's not Code Freeze?
+>
+> Absolutely, but it's not easy to enforce without hurting development of non-violators.
+
+@jberkus:
+
+> Shutting down merges is the nuclear option for preventing flakes and fails. We should be able to keep on top of them without resorting to that. But ... we're getting off topic here, unless folks think the "increased time to flake" is a blocker for this (I don't).
+
+#### TODO Attention to dependencies
+
+@jberkus:
+
+> Extra problems with upstream dependency patch support if our timing is bad
+
+## Design Details
+
+<!--
+This section should contain enough information that the specifics of your
+change are understandable. This may include API specs (though not always
+required) or even code snippets. If there's any ambiguity about HOW your
+proposal will be implemented, this is the place to discuss them.
+-->
+
+### TODO Schedule
```

Comment:
Read through the comments to collect the input.I think we can layout a lightweight policy, for example:

1. Always start the first release at the beginning of a year (ideally not the first week of January).
2. Always end the last release by mid December.
3. Choose a cycle length of ~15 weeks.
4. Consider KubeCon as a full week where no development and decision making happens.